x-pack/legacy/plugins/infra/public/containers/logs/log_highlights/log_highlights.tsx

```diff
@@ -22,14 +23,14 @@ export const useLogHighlightsState = ({
   sourceVersion: string | undefined;
 }) => {
   const [highlightTerms, setHighlightTerms] = useState<string[]>([]);
-
+  const [
+    {
+      logEntries: { entriesStart: startKey, entriesEnd: endKey },
```

Comment:
I'd either rename `entriesStart` as `startKey` or replace all `startKey` with `entriesStart`. Keeping a consistent name makes it easier to follow what's going on